src/rideshare/settings.py

```diff
@@ -28,7 +28,7 @@
 
 ALLOWED_HOSTS = environ.get("DJANGO_ALLOWED_HOSTS").split(",")
 
-POSTGRES = environ.get("POSTGRES")
+POSTGRES = False # TODO: change
```

Comment:
Why is it no longer an environment variable?

Reply:
sorry, Kate changed it when debugging and I forgot to change it back. had some issues because we weren't converting str to bool